AnnoyancesFilter/Cookies/sections/cookies_specific.txt

```diff
@@ -13035,6 +13035,9 @@ m.timesofindia.com,m-timesofindia-com.cdn.ampproject.org$$amp-consent
 ||autocentrum.pl/$replace=/(\n|"use strict";)\((window\._|self\.webpackChunk)_proto_[\s\S]*?"AKCEPTUJ.[\s\S]*?I PRZECHODZ.[\s\S]*?DO SERWISU".*//,script
 ||embed.dugout.com/v*/wpmedia.js$replace=/((.|this)\.get_gdpr\((.|this)\.run\.bind\((.|this)\)\))/\$1\,\$2\.run\.call\(\$2\)/,domain=sportowefakty.wp.pl
 ||embed.dugout.com/v*/wpmedia.js$replace=/.\.get_gdpr\(\(function\(\)\{return (.\.run\(\))\}\)\)/\$1/,domain=sportowefakty.wp.pl
+!
+! https://github.com/AdguardTeam/AdguardFilters/issues/177450
+shop.blu-blumen.de#%#//scriptlet('set-constant', 'cb_isMarketingAllowed', 'noopFunc')
```

Comment:
I think that it can be added without a comment, somewhere here - https://github.com/AdguardTeam/AdguardFilters/blob/master/AnnoyancesFilter/Cookies/sections/cookies_specific.txt#L267-L270

Reply:
done